content/members/members.json

```diff
@@ -411,5 +411,12 @@
     "github": "https://github.com/PaarthAgarwal",
     "link": "https://www.linkedin.com/in/paarth-agarwal-2291911b8/",
     "year": 2020
+  },
+  {
```

Comment:
Remove this change. Will add the members later.

Reply:
Bhaiya, I have removed this change